<!DOCTYPE html>
<html><head><title>GetDataFeed</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">5 references to GetDataFeed</div><div class="rA">Azure.AI.MetricsAdvisor (1)</div><div class="rG" id="Azure.AI.MetricsAdvisor"><div class="rF"><div class="rN">MetricsAdvisorAdministrationClient.cs (1)</div>
<a href="../MetricsAdvisorAdministrationClient.cs.html#337"><b>337</b>var createdDataFeed = <i>GetDataFeed</i>(dataFeedId, cancellationToken);</a>
</div>
</div>
<div class="rA">Azure.AI.MetricsAdvisor.Tests (4)</div><div class="rG" id="Azure.AI.MetricsAdvisor.Tests"><div class="rF"><div class="rN">MetricsAdvisorAdministrationClient\DataFeedTests.cs (4)</div>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DataFeedTests.cs.html#141"><b>141</b>Assert.That(() =&gt; adminClient.<i>GetDataFeed</i>(null), Throws.InstanceOf&lt;ArgumentNullException&gt;());</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DataFeedTests.cs.html#142"><b>142</b>Assert.That(() =&gt; adminClient.<i>GetDataFeed</i>(&quot;&quot;), Throws.InstanceOf&lt;ArgumentException&gt;());</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DataFeedTests.cs.html#143"><b>143</b>Assert.That(() =&gt; adminClient.<i>GetDataFeed</i>(&quot;dataFeedId&quot;), Throws.InstanceOf&lt;ArgumentException&gt;().With.InnerException.TypeOf(typeof(FormatException)));</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DataFeedTests.cs.html#155"><b>155</b>Assert.That(() =&gt; adminClient.<i>GetDataFeed</i>(FakeGuid, cancellationSource.Token), Throws.InstanceOf&lt;OperationCanceledException&gt;());</a>
</div>
</div>
</body></html>